Assets/Prefabs/Manager/ManagerGameObject/Manager.cs

```diff
@@ -277,8 +282,13 @@ public void Update()
     {
         if (started)

```

Comment:
Can change to
`if (!started) return;`
to reduce indentation

Reply:
Not mine